### PR TITLE
Add experience CRUD; other tiny fixes

### PR DESCRIPTION
**Pull request by henrymcarvajal-galilei**
- Navigation back to experiences list after successful creation is broken atm. Will review after some deserved rest. 😝 
- Otherwise, the feature is rather complete.
- Will integrate with Statistics views/components in other feature.

### CODE REVIEW

File: web/src/views/experience/questions/questionDetails.js
Comment (FernandoArevaloE) [inline web/src/views/experience/questions/questionDetails.js:1]: since this file is empty, is really needed?

Comment (henrymcarvajal-galilei) [reply web/src/views/experience/questions/questionDetails.js:1]: Will remove it on the next push 👍

File: app/src/db/experience.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const getExperienceById = async (id) => {
 }
 
 const listByCompanyId = async (companyId) => {
-  const res = await pool.pool.query('SELECT e.* FROM experience e JOIN actor a ON e.actor_id = a.actor_id WHERE a.company_id = $1', [companyId]);
+  const res = await pool.pool.query('SELECT e.experience_id, e.name, jsonb_array_length(e.questions) as questions, e.created_at, e.active FROM experience e WHERE e.company_id = $1', [companyId]);
```
Comment (andymotagl) [inline app/src/db/experience.js:9]: Could you please update the API documentation?

File: app/src/routes/jackpot.js [hunk 2]
```diff
@@ -11,6 +11,12 @@ router.get('/:id', auth('read:jackpots'), (req, res) =>
         .catch(error => handleError(error, res))
 )
 
+router.get('/', auth('read:jackpots'), (req, res) =>
```
Comment (andymotagl) [inline app/src/routes/jackpot.js:14]: I guess this requires to be added to the documentation

File: web/src/views/experience/components/QuestionsTable.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+import { v4 as nextId } from 'uuid';
+
+const QuestionsTable = ({ questions, setQuestions }) => {
+
+    const MAX_OPTIONS = 5;
+
+    const addQuestion = () => {
+        const newQuestion = {
+            id: nextId(),
```
Comment (andymotagl) [inline web/src/views/experience/components/QuestionsTable.js:14]: when you display it is hard to understand why the `3ag3d772-c975-401a-9944-9ad6b12d8c6f` is the correct and not the `e99cd16a-f73b-442c-91c2-96a8b34e3bf2`

**Review by andymotagl — COMMENTED** [review]
WHat a beautiful implementation! Now we should refactor everything else to look like this!
Some comments.

File: app/src/routes/jackpot.js (hunk 2 above)
Comment (henrymcarvajal-galilei) [reply app/src/routes/jackpot.js:14]: This was a change by mistake. Left previous version.

File: app/src/db/experience.js (hunk 1 above)
Comment (henrymcarvajal-galilei) [reply app/src/db/experience.js:9]: Documentation updated!

File: web/src/views/experience/components/QuestionsTable.js (hunk 3 above)
Comment (henrymcarvajal-galilei) [reply web/src/views/experience/components/QuestionsTable.js:14]: There is no need to display the id of the valid option to the user, The UI currently marks the option as correct in the options section.

Comment (henrymcarvajal-galilei) [reply web/src/views/experience/components/QuestionsTable.js:14]: I mean this id is not meant to be displayed on the UI itself for the user to see, but to mark the correct answer for the user. I can change back to a single digit, but it won't ever be displayed on the UI. Agree?

Comment (andymotagl) [reply web/src/views/experience/components/QuestionsTable.js:14]: Yeah, I agree with that and the id was never intended to be displayed in the UI but it was thought to be a simple digit number due the amount of questions and amount of options that would be created.

**Review by andymotagl — APPROVED** [review]

**Pull request merged**